tests/test_integrated_channels/test_sap_success_factors/test_transmitters/test_content_metadata.py

```diff
@@ -73,6 +78,10 @@ def test_transmit_create_failure(self, update_content_metadata_mock):
                 content_id: ContentMetadataItemExport(
                     {'key': content_id, 'content_type': 'course'},
                     {'courseID': content_id, 'update': True}
+                ),
+                content_id_2: ContentMetadataItemExport(
+                    {'key': content_id_2, 'content_type': 'course'},
+                    {'courseID': content_id_2, 'update': True}
```

Comment:
can you add a line to this test below that also asserts that there is no transmission object for content_id_2?

Reply:
Done